### DISCUSSION TITLE
Artifact naming questions: Name with spaces

### DISCUSSION THREAD

**Discussion by stdedos** [goreleaser/goreleaser#5007] [Q&A] [answered]
Why my artifacts have a space in them? (`junit2html_ SNAPSHOT...`)

https://github.com/stdedos/junit2html/actions/runs/9973285305/job/27558289659#step:6:60

https://github.com/stdedos/junit2html/commit/2246b216003cc833a10003a20a59e05f9433a4ef

**Comment by caarlos0** [comment, marked as answer]
```
      {{ .ProjectName }}_
```

should be 

```
      {{- .ProjectName }}_
```

https://github.com/stdedos/junit2html/blob/15ed8b7d7cd2e57838dfc8a2f9158cf1f474e576/.goreleaser.yaml#L18

**Comment by stdedos** [comment]
I'll... highjack my discussion, since I have more naming-related questions:

Can I has `.Version`, but without the `v` stripped?

If you check above, I was "medicating" that via `.Tag`.
However, I discovered that `snapshot.name_template` "overrides" the `.Version`.

So, ... this way (using `.Tag`) I'll need to fill my config with `{{ if .IsSnapshot }}SNAPSHOT-{{ .Summary }}_{{ else }}v{{ .Tag }}{{ end }}` (and "give up" the `snapshot.name_template`)

**Reply by caarlos0** [reply]
you could do `{{ trimprefix .Version "v" }}`

**Reply by stdedos** [reply]
(_split to https://github.com/orgs/goreleaser/discussions/5015; apologies, I thought you didn't see "after-answered" updates_ 😅)

**Comment by stdedos** [comment]
... can `snapshot.name_template` be renamed to ... `version_template`, or something like that?
(If my assumption above https://github.com/orgs/goreleaser/discussions/5007#discussioncomment-10074668 is correct ofc)

**Reply by caarlos0** [reply]
#5017

**Reply by stdedos** [reply]
(_split to https://github.com/orgs/goreleaser/discussions/5016_)
